client/gutenberg/extensions/publicize/publicize-form.jsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * Higher Order Publicize sharing form composition.
+ *
+ * Uses Gutenberg data API to dispatch publicize form data to
+ * editor post data in format to match 'publicize' field
+ * schema defined in {@see class-jetpack-publicize-gutenberg.php}
+ *
+ * @since  5.9.1
+ */
+
+/**
+ * External dependencies
+ */
+import { compose } from 'redux';
+import isNil from 'lodash/isNil';
+
+/**
+ * Internal dependencies
+ */
+const {
+	withSelect,
+	withDispatch,
+} = window.wp.data;
```

Comment:
this shouldn't reference `window` here - `= wp.data` or better `@wordpress/data`

Reply:
Fixed in bc312fd856b474c1b377c55d2a5192fb4692dc02